src/coreclr/jit/codegenarm64test.cpp

```diff
@@ -6946,6 +6946,60 @@ void CodeGen::genArm64EmitterUnitTestsSve()
     // IF_SVE_JK_4B
     theEmitter->emitIns_R_R_R_R(INS_sve_st1b, EA_SCALABLE, REG_V6, REG_P3, REG_R0, REG_V4,
                                 INS_OPTS_SCALABLE_D); // ST1B    {<Zt>.D }, <Pg>, [<Xn|SP>, <Zm>.D]
+
+    // IF_SVE_ID_2A
+    theEmitter->emitIns_R_R_I(INS_sve_ldr, EA_SCALABLE, REG_P1, REG_R5, 0); // LDR     <Pt>, [<Xn|SP>{, #<imm>, MUL VL}]
```

Comment:
can you fix the formatting so the entire comment comes on same line?

Reply:
I'll try to format it differently by putting it one line above.